prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -290,45 +300,73 @@ extension_9:
         system_code: "v2-0123"
 
 extension_10:
+  condition: $code NOT_EQUALS S && $code NOT_EQUALS A && $code NOT_EQUALS R && $code NOT_EQUALS U
   generateList: true
   expressionType: nested
+  vars:
+    code: String, TQ1.9.1
   expressionsMap:
     url:
       type: SYSTEM_URL
-      value: start-of-service
-    valueDateTime:
-      type: STRING
-      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
-      expressionType: JEXL
+      value: service-priority
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept_var
+      expressionType: resource
       vars:
-        dateTimeIn: TQ1.7
+        system: SYSTEM_URL, $system_code
+      constants:
+        system_code: "v2-0485"
 
 extension_11:
   generateList: true
   expressionType: nested
   expressionsMap:
     url:
       type: SYSTEM_URL
-      value: end-of-service
+      value: order-effective-date
     valueDateTime:
-      type: STRING
-      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
-      expressionType: JEXL
-      vars:
-        dateTimeIn: TQ1.8
+        type: STRING
+        valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
+        expressionType: JEXL
+        vars:
+          dateTimeIn: ORC.15
 
-extension_12:
-  generateList: true
-  expressionType: nested
-  expressionsMap:
-    url:
-      type: SYSTEM_URL
-      value: service-priority
-    valueCodeableConcept:
-      valueOf: datatype/CodeableConcept_var
-      expressionType: resource
-      vars:
-        system: SYSTEM_URL, $system_code
-        code: String, TQ1.9.1
-      constants:
-        system_code: "v2-0485"
\ No newline at end of file
+priority_1:
```

Comment:
A norm we've had so far has been to put the extensions all the way at the bottom of the file. It makes it easier to distinguish what is a part of the spec and what had to be added as an extension.

Reply:
Moved priority up to leave extensions at the bottom. Thanks for pointing that out.